frameworks/urls

```diff
@@ -1,9 +0,0 @@
-https://github.com/its-a-feature/Apfell
-https://github.com/cobbr/Covenant
-https://github.com/BishopFox/sliver
-https://github.com/FactionC2/Faction
-https://github.com/byt3bl33d3r/SILENTTRINITY
-https://github.com/nettitude/PoshC2_Python
-https://github.com/yeyintminthuhtut/Awesome-Red-Teaming
-https://github.com/Ne0nd0g/merlin
-https://github.com/zerosum0x0/koadic
```

Comment:
Please add these urls back, so they can be used in automation (specifically #20)